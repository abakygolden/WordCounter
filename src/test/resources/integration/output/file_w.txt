WANT 1
WARM 1
WE 5
WHITE 1
WITH 3
WITHOUT 1
WORD 1
WOULD 1