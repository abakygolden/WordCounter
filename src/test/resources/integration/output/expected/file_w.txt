WATERMELON 4
WINGS 4
WRAP 20